src/scores/continuous/standard_impl.py

```diff
@@ -5,6 +5,8 @@
 from typing import Optional
 
 import xarray as xr
+import numpy as np
```

Comment:
My initial feeling is that I'd prefer to let users convert their own lists, but I'd be happy to be convinced otherwise. Often, showing an example in a notebook makes it clearer whether a piece of functionality should be on the user side or the scores side.